gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html

```diff
@@ -4,21 +4,27 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>GPII Canopy MatchMaker tests</title>
 
-        <link rel="stylesheet" media="screen" href="../../../../../../node_modules/infusion/tests/lib/qunit/css/qunit.css" />
+        <link rel="stylesheet" media="screen" href="/node_modules/infusion/tests/lib/qunit/css/qunit.css" />
```

Comment:
I relativised all paths and confirmed that the Vagrant browser tests still pass.  I also confirmed that the following works:

1. From the root of the universal repo, run `python -m SimpleHTTPServer 8000`.
2. Open `http://localhost:8000/gpii/node_modules/canopyMatchMaker/test/web/html/CanopyMatchMakerUtilitiesTests.html` in a browser.
3. Confirm all tests pass.

@cindyli, I assume that the tests will now work with your setup as well.